scripts/icons-collect.mjs

```diff
@@ -54,12 +54,13 @@ const writeIconIndex = (outputFolder, iconName) => {
   const file = `${outputFolder}/index.tsx`
   const fileContent = `import React from 'react'
 
-import { Icon, IconProps } from '../../Icon'
+import { Icon } from '../../Icon'
+import type { IconProps } from '../../Icon.types'
 
 import content from './content.json'
 
 export const ${iconName}Icon: React.FC<IconProps> = props => {
-  return <Icon data-wui-icon alt="${iconName}" content={content} {...props} />
+  return <Icon alt="${iconName}" content={content} {...props} />
 }
 `
```

Comment:
Really non blocker but for the button we used a different syntax for typing the component:
```
import { Icon } from '../../Icon'
import type { IconProps } from '../../Icon.types'

import content from './content.json'

export const ActionsIcon = (props: IconProps) => {
  return <Icon alt="Actions" content={content} {...props} />
}
```